qemu/tests/rh_kselftests_vm.py

```diff
@@ -0,0 +1,58 @@
+from avocado.core import exceptions
+
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    rh_kselftests_vm test
+    1) Download the current kernel selftests RPM
+    2) Install the RPM
+    3) Execute the kernel selftests
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    session = vm.wait_for_login()
+    kernel_path = params.get("kernel_path", "/tmp/kernel")
+    tests_execution_cmd = params.get("tests_execution_cmd")
+    whitelist = params.get("whitelist").split()
+
+    session.cmd("mkdir -p %s" % kernel_path)
+    kernel_version = session.cmd_output("uname -r").strip().split("+")[0]
+    error_context.base_context("The kernel version: %s" % kernel_version, test.log.info)
+
+    error_context.context("Download the kernel selftests RPM", test.log.debug)
+    session.cmd("cd %s" % kernel_path)
+    session.cmd(
+        "brew download-build --rpm kernel-selftests-internal-%s.rpm" % kernel_version,
+        180,
+    )
+
+    error_context.context("Install the RPM", test.log.debug)
+    session.cmd("dnf install -y ./kernel-*")
+
+    try:
+        error_context.base_context("Execute the selftests", test.log.info)
+        s, o = session.cmd_status_output(tests_execution_cmd, 180)
+        test.log.info("The selftests results: %s" % o)
+
+        skipped_tests = True if "[SKIP]" in o else False
+        test.log.debug("Skipped tests: %r" % skipped_tests)
+        for test_name in whitelist:
+            if skipped_tests and test_name in o:
```

Comment:
test_name is always in the `o`,  no matter the status is pass, skip or fail.
You need to parser the output and get the skiped case name.

Reply:
Done !